### ISSUE TITLE
the fallback beacon node was considered to be unworkable, but it works well

### ISSUE THREAD

**Issue by winlin** [sigp/lighthouse#2140]
## Description

I set --beacon-nodes with two endpoints, for example:
```
http://a.a.a.a:5052,http://b.b.b.b:5052
```
when the `a` shutdown, but `b` works well, the validate-node consider it unusable but `b` works well, I can 
`telnet b.b.b.b:5052`
and get 200 response with :

```
curl http://b.b.b.b:5052/eth/v1/node/version

{"data":{"version":"Lighthouse/v1.0.6-f183af2/x86_64-linux"}}
``` 

```
CRIT Error during attestation routine        slot: 231054, committee_index: 1, error: "All endpoints failed http://a.a.a.a:5052/ => RequestFailed(\"Failed to produce an aggregate attestation: Reqwest(reqwest::Error { kind: Request, url: Url { scheme: \\\"http\\\", host: Some(Ipv4(a.a.a.a)), port: Some(5052), path: \\\"/eth/v1/validator/aggregate_attestation\\\", query: Some(\\\"slot=231054&attestation_data_root=0xc8f1e9bd4b0720ee962a361e84316f608d38c945cd0fbbc3f3f0e035b6cf40ec\\\"), fragment: None }, source: hyper::Error(Connect, ConnectError(\\\"tcp connect error\\\", Os { code: 111, kind: ConnectionRefused, message: \\\"Connection refused\\\" })) })\"), http://b.b.b.b:5052/ => RequestFailed(\"No aggregate available for AttestationData { slot: Slot(231054), index: 1, beacon_block_root: 0x49969fe83e3ba1a1ed91e4ca2391f35f345b7340e620f28598973bd69b938206, source: Checkpoint { epoch: Epoch(7219), root: 0xf14e95c11364c46cb37762143f84daef5cf41c95e508372c7ea891359d0cc8c4 }, target: Checkpoint { epoch: Epoch(7220), root: 0x3b641034a8c3802ce886cfd6332f5b8ad536c7e011042686583e521b30af75f7 } }\")", service: attestation
```

## Version

v1.0.6

**Comment by paulhauner** [comment]
Thanks for raising this @winlin.

The error for `b` is:

```
 http://b.b.b.b:5052/ => RequestFailed(\"No aggregate available for AttestationData
```

This means that the `b` node is reachable, but it didn't have an attestation available for aggregation. This is expected behaviour when the `--subscribe-all-subnets --import-all-attestations` flags are not provided to `lighthouse bn`.

I've been planning to write up some docs explaining this, I'll do that today or tomorrow and link them back here.

In the short term, use those two flags on the backup node (`b`). E.g.,

```
lighthouse bn --staking --subscribe-all-subnets --import-all-attestations
```

**Comment by winlin** [comment]
Very nice. Thanks @paulhauner .  It should be noted, we take the two validator node as the backup for each other, that mean `a` will be the backup node for `b`. so can we add `--subscribe-all-subnets --import-all-attestations` on `a` too ?

**Comment by paulhauner** [comment]
Yep, the only cost of those two flags is resource consumption. It'll cause the nodes to process all attestations on all 64 subnets, instead of ~1 per validator. The resource usage increase is roughly 2x, but it varies.

**Comment by winlin** [comment]
yes, you are right. after we add these two flags, the CPU usage increase from 6% into 12%.

**Comment by paulhauner** [comment]
> yes, you are right. after we add these two flags, the CPU usage increase from 6% into 12%.

Thank you for the feedback @winlin, much appreciated.

Please find my (draft) notes on the topic here: https://github.com/sigp/lighthouse/pull/2142/files

**Comment by winlin** [comment]
I have read the note, very clear. Good work.

**Comment by paulhauner** [comment]
Resolved in #2142
